forch/forch_metrics.py

```diff
@@ -75,6 +76,11 @@ def _add_vars(self):
                       'No. of RADIUS query timeouts in state machine', Counter)
         self._add_var('radius_query_responses',
                       'No. of RADIUS query responses received from server', Counter)
+        self._add_var('faucet_process_state', 'Current process state of Faucet', Gauge)
```

Comment:
this should be done with labels. The set of processes that's monitored is configurable in forch.yaml so it can't be hardcoded.

Reply:
Addressed.